tests/setup.js

```diff
@@ -0,0 +1 @@
+jest.setTimeout(10000000);
```

Comment:
What's this for?

Reply:
Debugging! Jest times out after 10s otherwise, killing the process and making it's impossible to debug without. But maybe it shouldn't be committed?

Reply:
Maybe just add a const explaining the magic number?